### PR TITLE
Prog/delete zeneszen bambusz gyogyfurdo shows

### PR DESCRIPTION
**Pull request by gammaw**
As requested by programming working group. Please review.

### CODE REVIEW

File: azuracast/liquidsoap_custom.liq [hunk 1]
```diff
@@ -340,7 +340,7 @@ sources = [
     ({3w and 18h-18h58m}, once(playlist_temporary_nites)),
     ({3w and 19h-19h58m}, once(playlist_rambo)),
     ({3w and 20h-21h58m}, once(playlist_random_times)),
-    ({3w and 15h30m-23h30m}, playlist_off_air_ambient),                     #ambient: EH - Gestalt
+    ({3w and 15h30m-21h30m}, playlist_off_air_ambient),                     #ambient: EH - Random Times
```
Comment (tuz666) [inline azuracast/liquidsoap_custom.liq:343]: The timeslot should be `12h30m-23h30m` and `Lahma Presents - Random Times`, nope?

**Review by tuz666 — APPROVED** [review]
I'm not sure about the OFF AIR times, but I had a comment about that, besides that it looks good to me!

File: azuracast/liquidsoap_custom.liq (hunk 1 above)
Comment (gammaw) [reply azuracast/liquidsoap_custom.liq:343]: No, check out the other days too. Our convention is that we play ambient music until half an hour into the last show. The rationale is that if it's a live show and starts a bit later (due to technical issues, delay etc.), we won't immediately play a banger sort of OFF AIR track, but more a transition/ambient kind of one. Makes sense? 

For the start, we don't want to play ambient in big gaps (between EH and Lahma presents), that's why we start with ambient when the "dense" afternoon program starts. 

@tuz666 please re-review.

Comment (tuz666) [reply azuracast/liquidsoap_custom.liq:343]: Yepp, I checked the other days, and the convention what I've seen was that the ambient playlist is starting half an hour later as the first show starts and it's ending 30 mins before the last show ends.
I was not aware of the gap after the show, I got your point now.

**Pull request merged**